web/ui/react-app/src/pages/starting/Starting.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC, useEffect } from 'react';
+import { RouteComponentProps, navigate } from '@reach/router';
+import { Progress, Alert } from 'reactstrap';
+import Logo from './Logo';
+
+import { useFetchReadyInterval, WalReplayData } from '../../hooks/useFetch';
+import { usePathPrefix } from '../../contexts/PathPrefixContext';
+
+interface StartingContentProps {
+  isResponding: boolean;
+  status?: WalReplayData;
+}
+
+export const StartingContent: FC<StartingContentProps> = ({ status, isResponding }) => {
+  if (!isResponding) {
+    return (
+      <Alert color="danger">
+        <strong>Error:</strong> Server is not responding
+      </Alert>
+    );
+  }
+
+  return (
+    <div className="text-center m-3">
+      <Logo height="400" />
```

Comment:
Hmm, it's a nice touch, but I'm not sure we should include a logo here, as we don't do that anywhere else. Of course, that's something we can think about separately some time on other pages, but let's not introduce it now just for a page that most people will never get to see.